src/widgets/msgbox/lv_msgbox.c

```diff
@@ -145,6 +146,7 @@ lv_obj_t * lv_msgbox_add_title(lv_obj_t * obj, const char * title)
         lv_obj_set_size(mbox->header, lv_pct(100), lv_display_get_dpi(lv_obj_get_display(obj)) / 3);
         lv_obj_set_flex_flow(mbox->header, LV_FLEX_FLOW_ROW);
         lv_obj_set_flex_align(mbox->header, LV_FLEX_ALIGN_START, LV_FLEX_ALIGN_CENTER, LV_FLEX_ALIGN_CENTER);
+        lv_obj_remove_flag(mbox->header, LV_OBJ_FLAG_SCROLLABLE);
```

Comment:
I don't think scroll for header and footer should be disable by default :thinking:

Reply:
Thank you for the review.

If the header of the msgbox is scrollable, 
I noticed that when accidentally touched directly on the header, 
it will have a similar appearance to this clip.

https://www.veed.io/view/047f1414-a8e6-4d1b-952d-c507a0349ff0?panel=quality-survey

I still recall instances where the header (or titlebar) had a scrollbar. 
Could you provide an example of a header (or titlebar) with a scrollbar?